grpcx/server.go

```diff
@@ -56,22 +58,78 @@ func WithRequestResponseLogs(l logx.Logger) grpc.UnaryServerInterceptor {
 
 // WithErrorLogs returns a gRPC interceptor for unary calls that instrument requests
 // with logs for the errors.
-func WithErrorLogs(l logx.Logger) grpc.UnaryServerInterceptor {
+func WithErrorLogs(l logx.Logger, options ...func(*WithErrorLogsOptions)) grpc.UnaryServerInterceptor {
+	var logOptions = &WithErrorLogsOptions{
+		debugLevelCodes: []codes.Code{},
+		infoLevelCodes:  []codes.Code{codes.Canceled, codes.Unknown, codes.InvalidArgument, codes.DeadlineExceeded, codes.NotFound, codes.AlreadyExists, codes.PermissionDenied, codes.ResourceExhausted, codes.FailedPrecondition, codes.Aborted, codes.OutOfRange, codes.Unimplemented, codes.Internal, codes.Unavailable, codes.DataLoss, codes.Unauthenticated},
```

Comment:
i don't think you need to have a list with all the possible codes. 
also what happens if they add a new one? you'll have to change this and that's a pita.
you should only need one list with the codes with debug level and the codes to discard.

Reply:
Ok, replaced by debugLevelCodes & discardedCodes